.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: triage
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Minimal steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues):
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
```

Comment:
A good idea could be to also ask them to run `next info`? (like in the [Next.js repo](https://github.com/vercel/next.js/blob/224447cd65d23f2dec2119b0c9ef10b79532dc6e/.github/ISSUE_TEMPLATE/1.bug_report.yml?plain=1#L51-L75))

Reply:
Very valid point.
At some point we should switch to the form version of the template.
I'll create an issue for that.